src/util/time.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Converts time to date (trims hours, mins, secs, ms)
+ */
+export function toDate(now: Date) {
+  const d = new Date(now)
+  d.setHours(0)
+  d.setMinutes(0)
+  d.setSeconds(0)
+  d.setMilliseconds(0)
+  return d
+}
+
+export function addOneWeek(d: Date) {
+  const ONE_WEEK = 7 * 24 * 60 * 60 * 1000
+  return new Date(d.getTime() + ONE_WEEK)
+}
```

Comment:
These functions already have their implementations in date-fns library (startOfDay and addWeeks, respectively). I think it's better to use ready-made solutions for this rather than implementing our own ones.

Reply:
Fixed in 19c05f5fbc61f6c5dc200438adc858c83f94dd3f